views/partials/participant-orders-list.html.twig

```diff
@@ -0,0 +1,43 @@
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <h3 class="panel-title"><i class="fa fa-medkit" aria-hidden="true"></i> Biobank Orders</h3>
+    </div>
+    <div class="panel-body">
+        {% for order in orders %}
+        {% if loop.index == 6 %}
```

Comment:
I like that you moved this into a partial view to reuse. When this is included from the order-modify.html.twig view, I think we want to show all orders, and not just the first 5 (otherwise, this would be confusing if the selected order is hidden). Maybe pass in an optional flag with the include to disable this div and the show all button. Also, can you fix the indentation here?

Reply:
Yes, I thought about it a while ago but some how missed to make that change. Will do that, thanks!